pkg/envoy/lds/connection_manager.go

```diff
@@ -17,13 +17,22 @@ import (
 	"github.com/openservicemesh/osm/pkg/featureflags"
 )
 
+// connectionDirection defines, for filter terms, the direction of the traffic from an application
+// perspective, in which the connection manager filters will be applied
+type connectionDirection string
+
 const (
 	meshHTTPConnManagerStatPrefix       = "mesh-http-conn-manager"
 	prometheusHTTPConnManagerStatPrefix = "prometheus-http-conn-manager"
 	prometheusInboundVirtualHostName    = "prometheus-inbound-virtual-host"
+
+	// incoming defines in-mesh inbound and ingress traffic driections
+	incoming = "incoming"
+	// outgoing defines in-mesh outbound and egress traffic directions
+	outgoing = "outgoing"
```

Comment:
For consistency with LDS naming conventions, s/incoming/inbound, s/outgoing/outbound.

Reply:
I thought we talked about this. I thought `inbound` is for in-mesh only, but here I needed a way to both identify `inbound` and `ingress`, hence the `incoming` tag. Lmk if you have other thoughts.

Reply:
I consider `inbound == mesh inbound || ingress`

Reply:
oh... ok, I'll change in this case

Reply:
done